server/src/utils/stats/CSVReader.cpp

```diff
@@ -0,0 +1,31 @@
+#include "CSVReader.h"
+
+namespace StatsUtils {
+    CSVTable readCSV(std::istream &istream, char sep, bool removeSpaces) {
```

Comment:
I changed type in CSVPrinter to char because in CSVReader we need to parse file by this separator, std::getline allows to provide a char separator, not a string